react-with-graphql-tutorial.md

```diff
@@ -1880,46 +1881,44 @@ const resolveAddStarMutation = mutationResult => state => {
+  * Align your final thoughts with {{% a_blank "this implementation" "https://github.com/rwieruch/react-graphql-github-vanilla" %}}.
+* Implement the `addReaction` mutation for an issue
+* Implement more fine-grained components (e.g. IssueList, IssueItem, ReactionList, ReactionItem)
+  * Extract components to their own files and use import and export statements to use them again in the App or other extracted components
 
 {{% chapter_header "Shortcomings of GraphQL in React without a GraphQL Client library" "react-graphql-library" %}}
 
-In the last sections, you have implemented a working yet simply GitHub issue tracker which uses React and GraphQL without a dedicated library for the GraphQL part. You have used axios to communicate with the GraphQL API with HTTP POST methods. Personally I believe it is important to work with raw technologies, in this case GraphQL by using plain HTTP methods, before introducing another abstraction on top of it. Especially when learning about those raw technologies in the first place. The Apollo library offers this abstraction that makes using GraphQL in React much easier. That's why you will use Apollo in a following section of this tutorial. For now, using GraphQL with only HTTP has shown you two important things before introducing Apollo for it:
+We implemented a simple GitHub issue tracker that uses React and GraphQL without a dedicated library for GraphQL, using axios to communicate with the GraphQL API with HTTP POST methods. I think it is important to work with raw technologies, in this case GraphQL, using plain HTTP methods, before introducing another abstraction.  The Apollo library offers an abstraction that makes using GraphQL in React much easier, so you will use Apollo in the nex section. For now, using GraphQL with HTTP has shown you two important things before introducing Apollo:
 
-* how does GraphQL work when using its puristic interface which which is nothing else than HTTP
-* what are the shortcomings of using no sophisticated GraphQL Client library in React
+* How GraphQL works when using its puristic interface which which is nothing else than HTTP.
+* The shortcomings of using no sophisticated GraphQL Client library in React.
 
-I want to address briefly the last fact in the list of important things you should have learned by now. What are the shortcomings of using puristic HTTP methods to read and write data to your GraphQL API in a React application? Before reading the following list, make up your mind on your own. Maybe you can contribute to the list your own thoughts later on. The following list states a couple of these shortcomings:
+I want to address the shortcomings of using puristic HTTP methods to read and write data to your GraphQL API in a React application. Before reading this list, make up your mind on your own. Maybe you can contribute to the list your own thoughts later on. The following list states a couple of these shortcomings:
```

Comment:
Remove: Maybe you can contribute to the list your own thoughts later on.

Reply:
We didn't need any of those latter sentences, really.  I think this is much more concise: "Before we move on, I want to address the shortcomings of using puristic HTTP methods to read and write data to your GraphQL API in a React application:" It goes straight into the list of shortcomings after saying you want to discuss them.